app/services/case_finder_service.rb

```diff
@@ -120,19 +122,20 @@ def in_states(states)
     scope.in_states(states.split(','))
   end
 
+  CASE_TYPES_ONLY_VISIBLE_WITH_FURTHER_CLEARANCE = [
```

Comment:
Not sure about what this is doing, could we put a comment in?

Reply:
I just moved it from a dynamic attribute to a static one whilst in the code. I don't really know any more